### DISCUSSION TITLE
My Game Still at 60FPS in my galaxy tab s8ultra android 14 one UI 6.0

### DISCUSSION THREAD

**Discussion by Nicarao1** [tribalfs/GalaxyMaxHzPub#236] [General]
I need help, I did buy this app but I have never get it to work, I do all the step by step that is on this web but my game still running 60fps.

When android 13 was I'd  use masfpspubg app and it work great, but now with this new android version (14, UI, 6.0)  NO APP WORK TO HAVE THIS TABLET AT 120 FPS.

Some help would be appreciated.

**Comment by tribalfs** [comment]
The app is not meant to control game fps but for display refresh rate (hz) on supported devices.
